src/components/tools/tool-tile.tsx

```diff
@@ -331,8 +304,12 @@ export class ToolTileComponent extends BaseComponent<IProps, IState> {
       return;
     }
 
-    const ToolComponent = kToolComponentMap[model.content.type].ToolComponent;
-    if (ToolComponent?.tileHandlesSelection) {
+    // Only select the tile if the tool doesn't handle it
+    // The name tileHandlesSelection is deceiving here it, it means the 
+    // tool doesn't handle it, so let the tile wrapper (this) handle 
+    // it.
+    const toolContentInfo = getToolContentInfoById(model.content.type);
+    if (toolContentInfo.tileHandlesSelection) {
```

Comment:
It's just a counter-intuitive name. How about `parentToolTileHandlesSelection` or something like that? I chose this polarity so that `undefined` would correspond to `false`, but clearly the name made more sense in my head at the time I wrote that code than it does to anyone else (or me) now.

Reply:
@kswenson, actually, what would think about reversing this? 

When making a basic tool, my intuition would be to let the framework handle the section by default. That way if the tool does nothing the selection would still work. 

So if a tool wants to do the extra work of managing its own selection then it would set this property.

In that case the name of the property could be:
`handlesOwnSelection`, `tileHandlesOwnSelection`, `toolHandlesOwnSelection`.  Or we could keep the same name as before `tileHandlesSelection` and reverse the meaning (probably the most confusing option for existing developers).

Reply:
I'm ok with that. I'd vote for `tileHandlesOwnSelection` for the name.

Reply:
done

Reply:
A questionable thing I did was in the placeholder tile to say that it handles its own selection.  If I understand it right, it should never be selected. So it "handles" it selection by not doing it. 😄 

However when I did a quick test it looked like I couldn't select the placeholder tile even if I didn't do this. I'm guessing there is some code to short circuit the selection if it is placeholder tile.